.prettierrc.json

```
@@ -0,0 +1,12 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
+  "importOrderSeparation": true,
+  "importOrderParserPlugins": ["typescript", "jsx"],
+  "printWidth": 80,
+  "bracketSpacing": true,
+  "trailingComma": "all",
+  "plugins": ["@trivago/prettier-plugin-sort-imports"]
```

Possibly needed here, too.
```suggestion
  "plugins": ["@trivago/prettier-plugin-sort-imports"]
  "htmlWhitespaceSensitivity": "ignore"
```

---

Thank you for mentioning that. I have searched for this option again and confirmed that it doesn't affect JSX code. It only affects [HTML template literals in JavaScript](https://prettier.io/blog/2018/11/07/1.15.0#html-template-literal-in-javascript). Therefore, we might not need to configure this option at the root level (for React).